pages/contact/_form.html

```diff
@@ -61,12 +41,12 @@
     <button
       type="submit"
       id="submit-button"
-      class="btn btn-primary"
+      class="col-xs-12 btn btn-primary"
```

Comment:
The long button looks a little weird. Perhaps the following would better?

<img width="1320" alt="image" src="https://user-images.githubusercontent.com/4295964/226067104-c93fbf05-bb6a-456c-b90f-a145d38a03a5.png">


If you like that idea, here's HTML to render it, but note that I have _not_ tested this code in any way.

```html
<div class="form-group">
    <div class="row">
      <div class="col-xs-12 col-md-4">
        <label for="contact-message"></label>
      </div>
      <div class="col-xs-12 col-md-8">
        <div class="text-center" style="text-align:right;display: block;">
    <br>
    <button type="submit" id="submit-button" class="col-xs-8 btn btn-primary" ga-on="click" ga-event-category="contact" ga-event-action="submit" style="
">
      Book a Demo
    </button>
  </div>
      </div>
    </div>
  </div>
```